test/UnitTest/Microsoft.AspNet.OData.Test.Shared/ODataStreamPropertyTest.cs

```diff
@@ -0,0 +1,222 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+#if NETCORE
```

Comment:
These seem duplicated

Reply:
what do you mean "duplicated"? @marabooy 

---
In reply to: [566650769](https://github.com/OData/WebApi/pull/2401#discussion_r566650769) [](ancestors = 566650769)

Reply:
I meant that the two blocks of the namespace imports are similar for both net core & non net core namespaces

Reply:
Yes. We separate them and avoid multiple #if #else #endif 

---
In reply to: [568498223](https://github.com/OData/WebApi/pull/2401#discussion_r568498223) [](ancestors = 568498223)